lua/markdown.lua

```diff
@@ -695,6 +750,16 @@ function M.toggle_checkbox()
     end
 end
 
+function M.reenum_wrapper(autocmd)
+    vim.schedule(function()
```

Comment:
I don't understand why it needs to be scheduled, can you leave a comment with a short explanation?

Reply:
I think this is because it needs to go 'edit -> schedule function -> update treesitter ast -> run scheduled and execute on new ast'
Without schedule it would do 'edit -> execute on old ast -> update ast'